ops/pebble.py

```diff
@@ -1969,6 +1986,88 @@ def read1(self, n: int = -1) -> Union[str, bytes]:
         return self.read(n)
 
 
+class IdentityAccess(enum.Enum):
```

Comment:
How about we make this a StrEnum, but that's not available on Python <3.10, so you can [do this](https://stackoverflow.com/questions/75040733/is-there-a-way-to-use-strenum-in-earlier-python-versions/75040847):

```suggestion
class IdentityAccess(str, enum.Enum):
```